samples/Shell/ash-env

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ash
+echo "ash"
```

Comment:
I can move both these and the previous ones (`bash` and `dash`) which I used as template to `tests/fixtures`. Although I'm not sure which test would be checking it?